src/main/java/com/arpnetworking/metrics/mad/Main.java

```diff
@@ -205,6 +206,9 @@ private void launchActors(final Injector injector) {
         // Create the telemetry connection actor
         actorSystem.actorOf(Props.create(Telemetry.class, injector.getInstance(MetricsFactory.class)), "telemetry");
 
+        // Create the actor sources supervisor actor
+        actorSystem.actorOf(SourceSupervisor.props(), "source");
```

Comment:
Will this supervise all source actors or will we need one per source actor? This pattern looks very familiar from CAgg but with some differences. Can you elaborate? (at least in pr, not necessarily in code).